BUILDING.md

```diff
@@ -300,53 +300,55 @@ To install this version of Node.js into a system directory:
 
 To verify the build:
 
-```console
-$ make test-only
+```bash
+make test-only
 ```
 
 At this point, you are ready to make code changes and re-run the tests.
 
-If you are running tests before submitting a pull request, the recommended
-command is:
+If you are running tests before submitting a pull request, use:
 
-```console
-$ make -j4 test
+```bash
+make -j4 test
```

Comment:
Switching to `bash` here does introduce some potentially confusing highlighting:

![image](https://user-images.githubusercontent.com/718899/143051413-9c8bb2da-e312-4237-8fc9-098170f1f1c1.png)

Back to `console` I guess. We might want to consider adding `script` to our allowed list as it will not highlight in surprising ways the way `bash` does, but doesn't semantically imply (to me at least) a prompt and output.